stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/database_logic.py

```diff
@@ -190,31 +180,53 @@ def apply_stacql_filter(search: Search, op: str, field: str, value: float):
         return search
 
     @staticmethod
-    def apply_sort(search: Search, field, direction):
+    def populate_sort(sortby) -> Optional[Dict[str, Dict[str, str]]]:
         """Database logic to sort search instance."""
-        return search.sort({field: {"order": direction}})
+        if len(sortby) > 0:
+            return {s.field: {"order": s.direction} for s in sortby}
+        else:
+            return None
 
     async def execute_search(
-        self, search, limit: int, base_url: str
-    ) -> Tuple[List[Item], Optional[int]]:
+        self,
+        search: Search,
+        limit: int,
+        token: Optional[str],
```

Comment:
now we need the token, and pass sort as a separate value instead of part of search